draft-ietf-rats-eat.md

```diff
@@ -223,32 +221,15 @@ CWT and JWT.
 
 # Introduction
 
-Some of the goals and fundamentals in the security model for attestation are not the same as other security standards such as those aimed at privacy (e.g., TLS) and authentication (e.g. FIDO).
-The security model for attestation is not described here.
-Instead see {{RATS.Architecture}}.
+The Entity Attestation Token (EAT) format enables entities to create EATs, which are attested claims sets
+that describe characteristics of the entity, i.e., a device, a software environment, a hardware environment or combination. The EAT format provides a common set of claims that can be used to represent evidence and attestation results. A relying party may use an EAT when determining the trustworthiness of the corresponding entity, and may choose to trust, not trust or partially trust the entity. An entity that verifies an EAT may create an EAT to convey attestation results for consumption by other entities.
 
-EAT provides the definition of a common set of claims that can be made about an entity, a device, some software and/or some hardware.
-This claims set is received by a relying party who uses it to decide if and how it will interact with the remote entity.
-It may choose to not trust the entity and not interact with it.
-It may choose to trust it.
-It may partially trust it, for example allowing monetary transactions only up to a limit.
+The EAT format defines the encoding of claims sets in both CBOR {{RFC8949}} and JSON {{RFC8259}} and defines EATs as extensions to the CBOR Web Token (CWT) {{RFC8392}} and JSON Web Token (JWT) {{RFC7519}} specifications. Authenticity and integrity protection MUST be provided and privacy (encryption) MAY additionally be provided for EATs. The CBOR Object Signing and Encryption (COSE) {{RFC9052}} and JSON Object Signing and Encryption (JOSE) {{RFC7515}} {{RFC7516}} SHOULD be used to provide these security services. Terminology, security models and operational models defined in [RATS.architecture] are used throughout.
 
-EAT defines the encoding of the claims set in CBOR {{RFC8949}} and JSON {{RFC7159}}.
-EAT is an extension to CBOR Web Token (CWT) {{RFC8392}} and JSON Web Token (JWT) {{RFC7519}}.
+The key material used to sign and encrypt SHOULD be created and provisioned exclusively for the purpose of attestation, as it is the use of this key material that makes the claims set "attested". [RATS.architecture] provides additional information related to provisioning key material used to sign and verify EATs.
```

Comment:
When you write "SHOULD" then you need to tell the reader when they should and should not create and provision the keying material exclusively for the purpose of attestation.

Also, how do you want to enforce this statement. I would turn this into a RECOMMENDED.

Reply:
I thought SHOULD and RECOMMENDED were more or less the same thing, but I see your point and think changing to "should" might be the right thing.